generator/src/main.rs

```diff
@@ -1330,8 +1330,18 @@ impl Parser {
                 pub #field_ident: pfn::#pfn_ident,
             };
             let c_name = c_name(name);
-            let init = quote! {
-                #field_ident: mem::transmute(entry.get_instance_proc_addr(instance, CStr::from_bytes_with_nul_unchecked(#c_name))?),
+
+            let init = if name != "xrEnumerateApiLayerProperties" {
+                quote! {
+                    #field_ident: mem::transmute(entry.get_instance_proc_addr(instance, CStr::from_bytes_with_nul_unchecked(#c_name))?),
+                }
+            }else {
```

Comment:
nit: `cargo fmt` please

Reply:
weirdly `cargo fmt` does not fix this.

Reply:
Huh. Sometimes it gives up if things get too complex, I guess we're hitting that.